doc/visual-programming/source/widgets/data/applydomain.md

```diff
@@ -1,4 +1,4 @@
-Transform
+Apply Domain
 =========
```

Comment:
Underscores have to fit the length of the title.

Reply:
Also, the change in not reflected in the index.rst. Could you please fix this too?

Reply:
Sorry for these continuous comments. The documentation for Apply Domain should be changed to reflect the changes in the widget name. Namely the references to the widget, image names (so we know what they refer to) and likely the example workflows.

Reply:
Done.
Apply Domain (ex Transform) not present in example workflows.

Reply:
Sorry, I didn't mean example workflows in Orange tutorial section, but workflows that are in Example section of the documentation.